get-parameter-table.py

```diff
@@ -0,0 +1,49 @@
+# Meant to be run from directory containing database
+
+import sys
+import nala
+nala.configure()
+from armi.bookkeeping import db
+
+def updateTableParameters(cycle, node, tableParameters, o=None):
+    if o is None:
+        o = db.loadOperator(casefile, cycle, node)
+    repInt = o.getInterface(name='report')
+    repInt.tableParameters = tableParameters.copy()
+    repInt.interactEveryNode(cycle, node)
+    tableParameters = repInt.tableParameters.copy()
+    return tableParameters
+
+"""
+Expects something like
+    python ../get-parameter-table.py refTestBase
+to be run from refTestBase/
+"""
+case = sys.argv[1]
```

Comment:
All this code can't be in the global scope. This gets run when people do `import armi`, which is not good.

Just encapsulate this logic in a function please.